testing/btest/core/suspend_processing/websocket.zeek

```diff
@@ -0,0 +1,102 @@
+# @TEST-DOC: Suspend pcap processing on a single-node worker, wait for a WebSocket client, resume processing and publish all new_connection() events on test.conns.{uid}
+#
+# @TEST-REQUIRES: python3 -c 'import websockets.sync'
+#
+# @TEST-PORT: WEBSOCKET_PORT
+#
+# @TEST-EXEC: cp $FILES/ws/wstest.py .
+#
+# @TEST-EXEC: zeek --parse-only ./worker.zeek
+#
+# @TEST-EXEC: btest-bg-run worker "zeek -r $TRACES/wikipedia.trace -b ../worker.zeek"
+# @TEST-EXEC: btest-bg-run client "python3 ../client.py"
+# @TEST-EXEC: btest-bg-wait 30
+#
+# @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-remove-abspath btest-diff worker/.stdout
+# @TEST-EXEC: TEST_DIFF_CANONIFIER=$SCRIPTS/diff-remove-abspath btest-diff client/.stdout
+#
+# @TEST-START-FILE worker.zeek
+
+global my_new_connection: event(uid: string, c: count);
+
+event zeek_init()
+	{
+	print network_time(), "zeek_init: suspend_processing()";
+	suspend_processing();
+	}
+
+event zeek_init() &priority=-5
+	{
+	Cluster::listen_websocket([
+		$listen_addr=127.0.0.1,
+		$listen_port=to_port(getenv("WEBSOCKET_PORT"))
+	]);
+	}
+
+event do_continue_processing() &is_used
```

Comment:
Is this used at all? From what I can tell, it doesn't seem to be called by anyone.